libs/service-portal/vehicles/src/module.tsx

```diff
@@ -36,7 +36,7 @@ export const vehiclesModule: PortalModule = {
       name: m.yourVehicles,
       path: VehiclePaths.AssetsMyVehicles,
       enabled: userInfo.scopes.includes(ApiScope.vehicles),
-      loader: translationLoader({ userInfo, client }),
+      loader: translationLoader({ userInfo, ...rest }),
```

Comment:
~Use this pattern for other modules where you used props instead of object destructuring?